packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1036,6 +1045,53 @@ export class Project extends ProjectBase {
     };
   }
 
+  private renderLogs(props: ProjectProps): CfnProject.LogsConfigProperty {
+    let s3Config: CfnProject.S3LogsConfigProperty|undefined;
+    let cloudwatchConfig: CfnProject.CloudWatchLogsConfigProperty|undefined;
+
+    if (props.logging?.s3) {
+      const s3Logs = props.logging.s3;
+
+      let status = 'ENABLED';
+      if ('enabled' in s3Logs) {
+        status = s3Logs.enabled ? 'ENABLED' : 'DISABLED';
+      }
```

Comment:
This code reads a little funny... can we simplify?

```ts
s3Config = {
  status: (s3Logs.enabled ?? true) ? 'ENABLED' : 'DISABLED'
  location: `${s3Logs.bucket.bucketName}${s3Logs.prefix}`,
  encryptionDisabled: s3Logs.encrypted ?? false,
}
```

Reply:
I like it